sample/Network/genesis_from_files.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python3
```

Comment:
this file is maintained in a different place, are you sure you want to make a second copy of it here and maintain it?